.gitignore

```diff
@@ -2,5 +2,3 @@ dist/
 vendor/
 .gh_token
 *.min.*
-templates/
```

Comment:
are you sure about this change ?
if yes, maybe it should be treated in another pr